tests/samples/PrivateMembers.scala

```diff
@@ -7,7 +7,22 @@ import scala.scalanative.native._
 @native.extern
 object PrivateMembers {
   type enum___privateEnum = native.CUnsignedInt
+  object enum___privateEnum {
+    final val A: enum___privateEnum = 0.toUInt
+    final val B: enum___privateEnum = 1.toUInt
+  }
+
   type enum_enumWithPrivateMembers = native.CUnsignedInt
+  object enum_enumWithPrivateMembers {
+    final val __C: enum_enumWithPrivateMembers = 0.toUInt
+    final val D: enum_enumWithPrivateMembers = 1.toUInt
+  }
+
+  object anonymous_enum_0 {
```

Comment:
Any specific reason why anonymous enums do not have the type alias?

Reply:
I wonder if it would even make sense to eventually just declare anonymous enum values unnested inside the main Scala object.

Reply:
Hmm, I think it will be better to generate type for anonymous enum, it will be more type safe then